app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/pocket/PocketStoriesComposables.kt

```diff
@@ -122,14 +126,14 @@ fun PocketStories(
     val flingBehavior = EagerFlingBehavior(lazyRowState = listState)
 
     LazyRow(
-        contentPadding = PaddingValues(start = 8.dp, end = 8.dp),
+        contentPadding = PaddingValues(horizontal = contentPadding),
         state = listState,
         flingBehavior = flingBehavior,
         horizontalArrangement = Arrangement.spacedBy(8.dp)
     ) {
         itemsIndexed(storiesToShow) { _, columnItems ->
-            Column {
-                columnItems.forEachIndexed { rowIndex, story ->
+            Column(verticalArrangement = Arrangement.spacedBy(8.dp)) {
```

Comment:
Ah yes, this is good stuff when we don't need that Spacer anymore.

Reply:
Elise saw this improvement. Indeed it's nice!